src/components/ResultsMap.vue

```diff
@@ -23,6 +31,7 @@
 				:icon="{ scaledSize: {width: 28, height: 45}, url: 'https://www.clker.com/cliparts/R/g/O/v/U/h/google-maps-marker-for-residencelamontagne-md.png' }"
 			/>
 		</GmapMap>
+		<results-list-item :result="this.selectedMarker"></results-list-item>
```

Comment:
Did we end up deciding to put this back underneath the map? I'm good either way.

Reply:
Yeah, when Nathan and I were running through the site, it looked really awkward at the top, so we moved it.

Reply:
Good to know! I guess sometimes we don't know what will look good until we get closer to the end of development.